Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyProductEvents.Codeunit.al

```diff
@@ -347,4 +347,21 @@ codeunit 30177 "Shpfy Product Events"
     internal procedure OnBeforeSendUpdateShopifyProductVariant(ShopifyShop: Record "Shpfy Shop"; var ShopifyVariant: Record "Shpfy Variant"; xShopifyVariant: Record "Shpfy Variant")
     begin
     end;
-}
\ No newline at end of file
+
+    [IntegrationEvent(false, false)]
```

Comment:
All new Events should include an xml documentation for any parameter.

Reply:
I agree fully on your suggestions! I have changed as according and committed the changes.